go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/redhatinsights/platform-go-middlewares v0.7.1-0.20201009171810-b73c54b47a2d
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/viper v1.7.1
+	github.com/therootcompany/xz v1.0.1 // indirect
```

Comment:
how did you pick this particular xz lib?

Reply:
I looked through a few xz decompression libraries.
[github.com/ulikunitz/xz](https://pkg.go.dev/github.com/ulikunitz/xz)'s description mentioned not being stable and having potential bugs.
[github.com/jamespfennell/xz](https://pkg.go.dev/github.com/jamespfennell/xz) seemed not to error handle in its NewReader function.
[github.com/therootcompany/xz](https://pkg.go.dev/github.com/therootcompany/xz) is marked as stable and returns the potential error from the NewReader function.

Reply:
@jharting after looking it a little closer, it looks like that lib is a fork of [https://github.com/xi2/xz](https://github.com/xi2/xz), so I've replaced it with the root repo.